### ISSUE TITLE
Accessibility: No way to escape chat using keyboard only

### ISSUE THREAD

**Issue by scottsweb** [Automattic/wp-calypso#44612]
Originally reported by @SiobhyB 

> It's not possible to navigate to the cross icon using keyboard only or to exit Happychat using the Esc key.

<img width="430" alt="b81c0014-6d53-11e6-8632-53faace682fe" src="https://user-images.githubusercontent.com/411945/89193709-af445580-d5a6-11ea-8530-ef721a25e513.png">

#### Steps to reproduce

1. Still trying to work out a good way to test this myself as Chat support seems difficult to trigger locally.
2.
3.
4.

#### What I expected

You should be able to close chat with your keyboard only

#### What happened instead

The `div` makes that impossible. 

#### Browser / OS version

All

#### Context / Source

Support chat repo.
